### PR TITLE
DO-238: Added dropdows for date fields

### PR DESCRIPTION
**Pull request by dhruvlahoti**

### CODE REVIEW

File: src/components/common/Typography/Form/FormFields/DatePickerInput/DatePickerInput.jsx [hunk 1]
```diff
@@ -56,15 +95,29 @@ const DatePickerInput = (props) => {
...
-        <Input
-          id={idSchema.$id}
-          style={{ ...fieldStyle }}
-          disabled={disabled}
-          readonly={readonly}
-          onChange={handleOnChange}
-          type="Date"
-          placeholder={placeholder}
-          value={formData || ''}
+        <DropdownDate
```
Comment (mohitsaladiya) [inline src/components/common/Typography/Form/FormFields/DatePickerInput/DatePickerInput.jsx:98]: No need to use library for this operation. Use alt-date widget of React-JsonSchema-Form.

File: package.json [hunk 2]
```diff
@@ -38,6 +38,7 @@
     "rc-upload": "^3.0.4",
     "react": "16.13.0",
     "react-dom": "16.13.0",
+    "react-dropdown-date": "0.0.18",
```
Comment (mohitsaladiya) [inline package.json:41]: This is not needed.

Comment (mohitsaladiya) [inline package.json:41]: This is not needed.

File: src/components/common/Typography/Form/FormFields/DatePickerInput/DatePickerInput.jsx (hunk 1 above)
Comment (dhruvlahoti) [reply src/components/common/Typography/Form/FormFields/DatePickerInput/DatePickerInput.jsx:98]: Will remove this is next PR.

**Pull request merged**